packages/builders/src/messages/embed/UnsafeEmbed.ts

```diff
@@ -126,7 +169,8 @@ export class UnsafeEmbed implements APIEmbed {
 	 * @param fields The fields to add
 	 */
 	public addFields(...fields: APIEmbedField[]): this {
-		this.fields.push(...Embed.normalizeFields(...fields));
+		if (!this.data.fields) this.data.fields = [];
+		this.fields?.push(...Embed.normalizeFields(...fields));
```

Comment:
Why is this using the `fields` getter? Also, should use the UnsafeEmbed class as to not perform validation
```suggestion
		this.data.fields ??= [];
		this.data.fields.push(...UnsafeEmbed.normalizeFields(...fields));
```